website/content/glossary/_index.md

```diff
@@ -0,0 +1,115 @@
+
+#### **Carbon Diet**
+
+Actions to reduce the output of CO2 to reduce the impact on the environment.
+
+
+#### **Carbon Footprint**
+
+Is the total amount of greenhouse gasses, often carbon dioxide, that a person, family, company, organization, or building (like a data center) releases to the environment. 
+This includes the release of GHG through direct use (e.g. the energy consumption of a data center) and indirect use (e.g. the energy required to produce servers).
+
+#### **Climate or Carbon Neutrality**
+
+Often used by organizations and companies to express that they equalize the cause of carbon pollution they generate. 
+This typically happens by buying negative carbon emissions from organizations that really saved carbon or are net-zero or even reduce more carbon than they produce. 
+However, it is important to understand that this is only a compensation and does not have to imply a new product development leading to more sustainability. Please also read about the term "Net Zero" as both terms are often mixed or misleadingly used.
+
+## E
+
+#### **Eco-Efficiency**
+
+Goods and services that use fewer resources and create less waste and pollution.
+
+#### **Emissions**
+
+An umbrella term under which for example also greenhouse gasses are covered. But emissions include other factors like micro particles that cause pollution. Emissions are categorized into three scopes:
+Scope 1: direct emissions: the CO2 output of the vehicles owned by the company
+Scope 2: indirect emissions: the power consumption of a data center, that causes emissions by the production of electricity
+Scope 3: indirect emissions, that not directly can be changed: delivery chain caused CO2 pollution
+
+
+#### **Embodied (Carbon) Emissions**
+
+Embodied, sometimes referred to as Embedded, Carbon Emissions are all emissions caused during a good's production. 
+That can include anything from transportation, storage, heating, the production of other goods to finish this good and so on.
+
+#### **Environmental Sustainability**
+
+Sustainability is the concept of a social, economical and environmental activity that, in simple terms, leaves the ecosystem as it found it. 
+Environmental sustainability focuses on the ecological impact of our doings and how we can improve our actions to reduce or eliminate our negative impact on nature, earth, and finally humanity.
+
+## G
+
+#### **Green House Gas (GHG)**
+
+Any kind of gasses caused by nature or human activities that trap heat in the atmosphere. GHG includes carbon dioxide, methane, ozone and others. 
+These gasses are called greenhouse gas because they have the same effect as a greenhouse: the sunlight can enter the house and part is reflected back, but it can‘t leave the greenhouse fully because the roof prevents the heat from escaping. 
+This causes the inside of a greenhouse to get hotter and hotter, the same as what’s going on with the earth. See also: Emissions.
```

Comment:
Style question:  Do we want to avoid contractions in our writing?  "what is" instead of "what's"?  Happy to be overruled, but when writing formal documents I generally avoid them.